test/e2e_federation/ingress.go

```diff
@@ -323,16 +323,17 @@ func waitForIngressOrFail(clientset *kubeclientset.Clientset, namespace string,
 	By(fmt.Sprintf("Fetching a federated ingress shard of ingress %q in namespace %q from cluster", ingress.Name, namespace))
 	var clusterIngress *v1beta1.Ingress
 	err := wait.PollImmediate(framework.Poll, timeout, func() (bool, error) {
-		clusterIngress, err := clientset.Ingresses(namespace).Get(ingress.Name, metav1.GetOptions{})
-		if (!present) && errors.IsNotFound(err) { // We want it gone, and it's gone.
+		var get_err error
+		clusterIngress, get_err = clientset.Ingresses(namespace).Get(ingress.Name, metav1.GetOptions{})
```

Comment:
I think there was no need to change the `err` variable name to get_err. please revert it.